### ISSUE TITLE
Asking for the dataset release 

### ISSUE THREAD

**Issue by YOONSEOKHEO** [ViCap01/ViCap#1]
First of all, I would like to say thank you for sharing your work on Github. 

I would like to adopt your work as a baseline for my ongoing research. 
While there are a train and a validation set for the pretraining phase, I cannot find a train/validation/test set for training a ViCap model. 
In the paper, you mentioned that for each clip of the validation and test set of ViCap models, you give three different semantic sentences as annotations. Could you please share those datasets to equally compare the performance?
